packages/pg-protocol/src/inbound-parser.test.ts

```diff
@@ -39,58 +39,58 @@ var twoRowBuf = buffers.rowDescription([
   },
 ])
 
-var emptyRowFieldBuf = new BufferList().addInt16(0).join(true, 'D')
+let emptyRowFieldBuf = new BufferList().addInt16(0).join(true, 'D')
 
-var emptyRowFieldBuf = buffers.dataRow([])
+emptyRowFieldBuf = buffers.dataRow([])
```

Comment:
The only manual change was here due to: cannot redeclare variable. Same at line 52.
I think the line 42 and 46 can be safely removed because they are declared twice but only the 2nd var is used.